src/alchemlyb/visualisation/dF_state.py

```diff
@@ -14,8 +14,8 @@
 
 from ..estimators import TI, BAR, MBAR
 
-def plot_dF_state(estimators, labels=None, colors=None, units='kcal/mol',
-                  orientation='portrait', nb=10):
+def plot_dF_state(estimators, labels=None, colors=None, units='kT',
+                  scaling_factor=1, orientation='portrait', nb=10):
```

Comment:
remove the scaling factor feature from this PR and make a new one where we can address unit conversion